src/main/java/RedisSessionManagerT8.java

```diff
@@ -0,0 +1,319 @@
+import org.apache.catalina.*;
+import org.apache.catalina.session.ManagerBase;
+import org.apache.catalina.session.StandardSession;
+import org.apache.catalina.util.LifecycleSupport;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+import org.apache.commons.lang3.SerializationUtils;
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+import org.apache.juli.logging.Log;
+import org.apache.juli.logging.LogFactory;
+
+/**
+ * Created by Cesar Valverde on 6/10/2015.
+ */
+public class RedisSessionManagerT8 extends ManagerBase implements Lifecycle {
+
+    private static final Log log = LogFactory.getLog(RedisSessionManagerT8.class);
+
+    private static final String REDIS_METADATA_KEY = ":metadata";
+    private static final String REDIS_ATTRIBUTES_KEY = ":attributes";
+
+    //Metadata keys
+    public static final String METADATA_VALID = "valid";
+    public static final String METADATA_CREATION_TIME = "creation_time";
+    public static final String METADATA_LAST_ACCESS_TIME = "last_access_time";
+    public static final String METADATA_MAX_INACTIVE_INTERVAL = "max_inactive_interval";
+
+    protected JedisPool redisConnectionPool = null;
+    protected JedisPoolConfig redisConnectionPoolConfig = null;
+
+    //Database info
+    private String dbHost = "localhost";
+    private int dbPort = 2222;
+    private int dbTimeout = 0;
+    private String dbPassword = "";
+
+    private static final String ISO_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ssZ";
```

Comment:
Also, note that you're hardcoding the date format to Zulu time/GMT, which is almost certainly wrong.